src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -496,6 +501,37 @@ theorem toInt_pos_iff {w : Nat} {x : BitVec w} :
     0 ≤ BitVec.toInt x ↔ 2 * x.toNat < 2 ^ w := by
   simp [toInt_eq_toNat_cond]; omega
 
+/-
+If `x.msb` is false, then the value of `x` when interpreted as a 2s complement
+integer is between `[0..2^n/2)`. 
+-/
+theorem toInt_bounds_of_msb_eq_false {x : BitVec n} (hmsb : x.msb = false) :
```

Comment:
@kim-em what do I name this theorem? I want to express that this gives us bounds on `x.toInt` when it is know that `x.msb = false`.

Reply:
A similar concern for `toInt_bounds_of_msb_eq_true`.